functional-tests/README.md

```diff
@@ -30,6 +30,13 @@ rm -rf bitcoin-28.0 bitcoin-28.0-x86_64-linux-gnu.tar.gz
 bitcoind --version
 ```
 
+On macOS, you must have Rosetta installed:
+
```

Comment:
I would love to have this explained. Maybe something like "for the Python dependencies that need to compile `solx`, macOS users need to activate rosetta emulation".

Reply:
Done!